### PR TITLE
Add `github` option for `--ci` in scaffold package tests

### PR DESCRIPTION
**Pull request by ernilambar**
Fixes: https://github.com/wp-cli/scaffold-package-command/issues/217

* Add `github` option for `--ci` argument in `scaffold package-tests`
* Two files are added for this CI (apart from common test files).
  - `behat.yml` - Same file from the root is copied
  - `testing.yml` - Template file is kept at `dot-github/workflows` folder. This will be copied to `.github/workflows`

### CODE REVIEW

**Review by wojtekn — APPROVED** [review]
The change looks clear, and works fine:

```
% vendor/bin/wp scaffold package-tests wp-cli-shim --ci=github 
Success: Created package test files.
```

**Review by danielbachhuber — APPROVED** [review]

File: dot-github/workflows/testing.yml [hunk 1]
```diff
@@ -0,0 +1,52 @@
+name: Testing
```
Comment (danielbachhuber) [inline dot-github/workflows/testing.yml:1]: This should live in `templates/`, no?

File: dot-github/workflows/testing.yml [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  behat:
+    strategy:
+      matrix:
+        php-version: ['8.0', '7.4']
```
Comment (danielbachhuber) [inline dot-github/workflows/testing.yml:19]: Let's add 8.2 here too

**Review by danielbachhuber — CHANGES_REQUESTED** [review]
Can you add some basic tests for this?

**Comment by ernilambar** [comment]
@danielbachhuber PR has been updated.

**Review by danielbachhuber — APPROVED** [review]
Thanks @ernilambar !

**Pull request merged**